static/scss/partials/onboarding.scss

```diff
@@ -393,29 +393,57 @@ firefox-private-relay-addon[data-addon-installed="true"] + firefox-private-relay
 }
 
 .c-premium-onboarding-action-completed {
-    padding: $spacing-md;
+    padding: $spacing-lg;
     padding-inline-end: $spacing-lg;
     background-color: $color-white;
     display: none;
     align-items: center;
     border-radius: $border-radius-md;
-    width: auto;
-
-    samp {
```

Comment:
Issue: If you switch the tag back to a `samp` tag, you may need to move some of this back.

Reply:
Switched back to a samp tag!